packages/fast-tooling-react/src/css-property-editor/property-editor-row.tsx

```diff
@@ -129,9 +128,7 @@ export default class CSSPropertyEditorRow extends Foundation<
      * a user to continue typing using spinal case
      */
```

Comment:
This comment isn't really accurate - do we even need this function anymore? Can we just parse the string with `spinalCase` prior to passing it into the input value?

Reply:
Good point, I'll remove